src/resources/PlatformResources.ts

```diff
@@ -40,6 +41,7 @@ const resourcesMap: Array<{key: string; resource: typeof Resource}> = [
     {key: 'search', resource: Search},
     {key: 'resourceSnapshot', resource: ResourceSnapshots},
     {key: 'ua', resource: UsageAnalytics},
+    {key: 'regionConfiguration', resource: RegionConfiguration},
```

Comment:
Do you think if it's better to use `regionConfig` since it's shorter? :sooptional: :stuck_out_tongue:

Reply:
I initially thought about shortening it, but then I decided to keep the same name as the real service to avoid any "confusion". TBH I don't mind either way.
![image](https://user-images.githubusercontent.com/35579930/86787989-aad25e80-c033-11ea-81b8-582a5759c86f.png)